extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -89,6 +100,27 @@ export class AmpExperiment extends AMP.BaseElement {
       return experiments;
     });
   }
+
+  /**
+   * Parses URL fragment and returns variant override.
+   * @return {!Object<string, string>}
+   * @private
+   */
+  getVariantOverride_() {
+    const queries = parseQueryString(
```

Comment:
We do it sometimes when we can't rely on viewer being initialized already. It looks something like: `(location.originalHash || location.hash)`. But why not just use `viewer.getParam` here?